src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -55,14 +55,35 @@
  * a different <a href="https://checkstyle.org/property_types.html#scope">scope</a>.
  * </p>
  * <p>
- * Violates parameters and type parameters for which no param tags are present
- * can be suppressed by defining property {@code allowMissingParamTags}.
- * Violates methods which return non-void but for which no return tag is present
- * can be suppressed by defining property {@code allowMissingReturnTag}.
- * Violates exceptions which are declared to be thrown, but for which no throws
- * tag is present by activation of property {@code validateThrows}.
+ * Violates parameters and type parameters for which no param tags are present can
+ * be suppressed by defining property {@code allowMissingParamTags}.
  * </p>
  * <p>
+ * Violates methods which return non-void but for which no return tag is present can
+ * be suppressed by defining property {@code allowMissingReturnTag}.
+ * </p>
+ * <p>
+ * Violates exceptions which are declared to be thrown (by 'throws' in the method
+ * signature or by 'throw new' in the method body), but for which no throws tag is
+ * present by activation of property {@code validateThrows}.
+ * ATTENTION: Checkstyle does not have information about hierarchy of exception types
```

Comment:
Should attention be separated from the rest. Site was not generated but I believe "ATTENTION" will appear mid-line and not at the start.

Reply:
Fixed @ https://github.com/checkstyle/checkstyle/pull/7955#discussion_r417078461